### PR TITLE
podman secret create -d alias --driver, inspect -f alias --format: Docker compatibity

### PR DESCRIPTION
**Pull request by rhatdan**
Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

#### Does this PR introduce a user-facing change?

```release-note
podman secret create now supports -d as alias for --driver: Docker Compatibility.
podman secret inspect supports -f as alias for --format: Docker Compatibility.

```

### CODE REVIEW

**Comment by mheon** [comment]
LGTM

File: test/e2e/secret_test.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ var _ = Describe("Podman secret", func() {
 		err := ioutil.WriteFile(secretFilePath, []byte("mysecret"), 0755)
 		Expect(err).To(BeNil())
 
-		session := podmanTest.Podman([]string{"secret", "create", "--driver-opts", "opt1=val", "a", secretFilePath})
+		session := podmanTest.Podman([]string{"secret", "create", "-d", "file", "--driver-opts", "opt1=val", "a", secretFilePath})
```
Comment (TomSweeneyRedHat) [inline test/e2e/secret_test.go:43]: I don't see a test using `--driver`, can you add one unless I missed it?

**Pull request merged**